.travis.yml

```diff
@@ -1,14 +1,13 @@
-# We want Ubuntu 14.04 "Trust" instead of 12.04 "Precise" to get newer
-# and less buggy gcov
-dist: trusty
+# Force Ubuntu 16.04 "Xenial" to get newer GCC, binutils etc.
+dist: xenial
 language: c
 
 env:
   global:
     - CFLAGS="--coverage -O2"
     - LDFLAGS="--coverage"
     # default config flags: enable debug asserts, and treat compiler warnings as errors
-    - CONFIGFLAGS="--enable-debug --enable-Werror"
```

Comment:
If we do not treat compiler warnings as errors (but why?) then the comment should be updated too. But I'd prefer to treat them as errors for the core GAP system, but not for packages.

Reply:
If you look at the commit messages, they explain that we actually keep using --enable-Werror, it simply is redundant to set this here again

Reply:
but yeah, the comment should be updated